hieradata/cluster/luan.yaml

```diff
@@ -1,4 +1,46 @@
 ---
 clustershell::groupmembers:
   luan: {group: "luan", member: "luan[01-05]"}
+profile::core::ospl::enable_rundir: true
+profile::core::k8snode::enable_dhcp: true
```

Comment:
This is only used for multus. We don't want it on luan.  The cni-dhcp service will need to be manually removed from nodes.